src/libraries/System.Text.Json/tests/Common/ConstructorTests/ConstructorTests.ParameterMatching.cs

```diff
@@ -930,7 +930,7 @@ public async Task ArgumentDeserialization_UseNamingPolicy_ToMatch_CaseInsensitiv
         {
             var options1 = new JsonSerializerOptions
             {
-                PropertyNamingPolicy = new SimpleSnakeCasePolicy()
+                PropertyNamingPolicy = JsonNamingPolicy.SnakeCaseLower
```

Comment:
Replaces a test class implementing snake case with the new snake case naming policy included with the product.